FSJ_django20_project/FSJ/views_coordinator.py

```diff
@@ -1 +1 @@
    FSJ_user = get_FSJ_user(request.user.username)
    try:
        student = Student.objects.get(ccid = usr_ccid)
    except Student.DoesNotExist:
        raise Http404("Student does not exist")
    
    if request.method == "POST":
        form = StudentForm(request.POST, instance=student)
        if form.is_valid():
            form.save()
            return redirect('studentlist')
        else:
            raise ValueError
    else:
        form = StudentForm(instance=student)
        context = get_standard_context(FSJ_user)
        context["student"] = student
        context["form"] = form
        url = "/studentlist/" + student.ccid + "/"
        context["url"] = url
        template = loader.get_template("FSJ/profile.html")
        return HttpResponse(template.render(context, request))


@login_required
@user_passes_test(is_coordinator)
def coordinator_adjudicatordetail(request, usr_ccid):
    FSJ_user = get_FSJ_user(request.user.username)
    try:
        adjudicator = Adjudicator.objects.get(ccid = usr_ccid)
    except Adjudicator.DoesNotExist:
        raise Http404("Adjudicator does not exist")
    
    if request.method == "POST":
        form = AdjudicatorForm(request.POST, instance=adjudicator)
        if form.is_valid():
            form.save()
            return redirect('adjudicatorlist')
        else:
            raise ValueError
    else:
        form = AdjudicatorForm(instance=adjudicator)
        context = get_standard_context(FSJ_user)
        context["adjudicator"] = adjudicator
        context["form"] = form
        url = "/adjudicatorlist/" + adjudicator.ccid + "/"
        context["url"] = url
        template = loader.get_template("FSJ/profile.html")
        return HttpResponse(template.render(context, request))

@login_required
@user_passes_test(is_coordinator)
def coordinator_addstudent(request):
    FSJ_user = get_FSJ_user(request.user.username)
    if request.method == "POST":
        form = StudentForm(request.POST)
        if form.is_valid:
            form.save()
            return redirect('studentlist')
    else:
        form = StudentForm()
           
    context = get_standard_context(FSJ_user)
    template = loader.get_template("FSJ/profile.html")
    context["form"] = form
    url = "/studentlist/add/"
    context["url"] = url
    return HttpResponse(template.render(context, request))

@login_required
@user_passes_test(is_coordinator)
def coordinator_addadjudicator(request):
    FSJ_user = get_FSJ_user(request.user.username)
    if request.method == "POST":
        form = AdjudicatorForm(request.POST)
        if form.is_valid:
            form.save()
            return redirect('adjudicatorlist')
    else:
        form = AdjudicatorForm()
           
    context = get_standard_context(FSJ_user)
    template = loader.get_template("FSJ/profile.html")
    context["form"] = form
    url = "/adjudicatorlist/add/"
    context["url"] = url
    return HttpResponse(template.render(context, request))

```

Comment:
Inconsistent indenting between add student/adjudicator versus edit student/adjudicator inside the if request.method == "POST" in the else block. 
See comment in views - profile re: raising value errors and incorporate a way to return the invalid form to the context.